### PR TITLE
DM-36925: add menu option to exclude products from tarball builds

### PR DESCRIPTION
**Pull request by mwittgen**

### CODE REVIEW

File: pipelines/release/official_release.groovy [hunk 1]
```diff
@@ -21,13 +21,15 @@ notify.wrap {
...
     'O_LATEST',
+    'EXCLUDE_FROM_TARBALLS'
   ])
 
   String sourceGitRefs = params.SOURCE_GIT_REFS
   String gitTag        = params.RELEASE_GIT_TAG
   String splenvRef     = params.SPLENV_REF
   String rubinEnvVer   = params.RUBINENV_VER
   Boolean dockerLatest = params.O_LATEST
+  String exclude       = params.EXCLUDE_FROM_TARBALLS,
```
Comment (ktlim) [inline pipelines/release/official_release.groovy:32]: But no comma is wanted here.

**Review by ktlim — CHANGES_REQUESTED** [review]
Aren't you missing the changes to `jobs/official_release.groovy`?

File: jobs/official_release.groovy [hunk 2]
```diff
@@ -24,6 +24,8 @@ p.pipeline().with {
     stringParam('SPLENV_REF', scipipe.template.splenv_ref, 'conda env ref or eups tag')
     stringParam('RUBINENV_VER', scipipe.template.splenv_ref, 'rubin-env version')
     booleanParam('O_LATEST', false,
-      'update the eups "O_LATEST" tag -- should only be done for a final (non-rc) release')
+      'update the eups "O_LATEST" tag -- should only be done for a final (non-rc) release'),
+    stringParam('EXCLUDE_FROM_TARBALLS', null,
+      'products to exclude from tarball builds. Eg. "lsst_sitcom"')
```
Comment (ktlim) [inline jobs/official_release.groovy:29]:
```suggestion
      'products to exclude from tarball builds, e.g. "lsst_sitcom"')
```

**Review by ktlim — APPROVED** [review]

File: jobs/official_release.groovy (hunk 2 above)
Comment (mwittgen) [reply jobs/official_release.groovy:29]: Updated all other occurrences to proper English

**Pull request merged**